packs/vuln-management.conf

```diff
@@ -42,31 +42,39 @@
       "query" : "select browser_plugins.* from users join browser_plugins using (uid);",
       "interval" : "86400",
       "platform" : "darwin",
-      "version" : "1.4.5",
+      "version" : "1.6.1",
       "description" : "Retrieves the list of C/NPAPI browser plugins in the target system.",
       "value" : "General security posture."
     },
     "safari_extensions": {
       "query" : "select safari_extensions.* from users join safari_extensions using (uid);",
       "interval" : "86400",
       "platform" : "darwin",
-      "version" : "1.4.5",
+      "version" : "1.6.1",
       "description" : "Retrieves the list of extensions for Safari in the target system.",
       "value" : "General security posture."
     },
+    "opera_extensions": {
+      "query" : "select opera_extensions.* from users join opera_extensions using (uid);",
+      "interval" : "86400",
+      "platform" : "posix",
+      "version" : "1.6.1",
+      "description" : "Retrieves the list of extensions for Opera in the target system.",
+      "value" : "General security posture."
+    },
     "chrome_extensions": {
       "query" : "select chrome_extensions.* from users join chrome_extensions using (uid);",
       "interval" : "86400",
       "platform" : "darwin",
```

Comment:
![grk1uzm](https://cloud.githubusercontent.com/assets/1271349/22897540/eb30a3d8-f1d8-11e6-8cd8-df59ba969b9f.gif)

I will update it accordingly! Thank you!